### ISSUE TITLE
Table: screen reader not announcing growingButtonText and growingButtonSubtext correctly

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#5439]
**Describe the bug**
For the Responsive `Table` component screen reader should announce the `growingButtonText` only. It currently announces unnecessary text e.g. `row 52 Name column 1 through 9 More button` when the actual text is `More`. It does not announce `growingButtonSubtext` at all. It should announce the value of `growingButtonSubtext` also.

**Expected behavior**
Responsive Table should provide a separate attribute for **screen readers only** to announce the user friendly text for `growingButtonText` and `growingButtonSubtext` attributes. if that attributed is not used, it should announce the default values of  `growingButtonText` and `growingButtonSubtext`. 

**Screenshots**
<img width="1905" alt="Screen Shot 2022-06-29 at 9 32 04 AM" src="https://user-images.githubusercontent.com/65705637/176490066-c040c122-66ca-4e7c-91f2-7155ccd9fd5b.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.4.0
`@ui5/webcomponents-react` version: 0.25.2

**Comment by niyap** [comment]
Hello Colleagues,

Thanks for sharing your findings!

Actually, that has never been implemented so we will handle it as a feature request as we have to check the actual behaviour in sap.m.Table as well as to discuss it with the accessibility experts.

Kind Regards,
Niya

**Comment by hristop** [comment]
Hi, 

The unnecessary text announcement will be handled with the growing button refactoring: #5813. 
What will be left here is the announcement of the growing button sub-text. 

Best Regards,
Hristo

**Comment by hristop** [comment]
Hi @gurkirpalgill 

We are writing to inform you about an important development regarding our ui5-table component. After careful consideration and received feedback, we have decided to focus on re-implementing and improving the existing table component based on the gathered feature requests and requirements.
Here are some details about this decision and how it may affect our ongoing development efforts.
 
The decision was made based on several factors, including scalability, performance, accessibility and the ability to meet future feature requirements. Our team has recognised the limitations of the current table component and after careful consideration, we concluded that it would be more effective to develop a new table component from scratch rather than invest more time in enhancing the current one.
 
While this decision allows us to provide a better user experience and meet evolving needs, I want to acknowledge that it may have some implications for the feature requests that have been made for the current ui5-table component. We understand that stakeholders have invested time and effort in suggesting improvements and new features. Rest assured that we value your input, and we will make every effort to take these feature requests into consideration during the development of the new ui5-table component.
 
However, it is important to note that the new table component may introduce backward incompatible changes. These changes may impact the way the current table component functions, including its API and visual representation. We understand that this may require adjustments and adaptations on your end. Our team will strive to provide documentation and guidance to facilitate a smooth transition to the new table component.
 
We are excited about the opportunities this transition will bring and the positive impact it will have on our product.
 
Should you have any questions or concerns regarding this transition or its implications, please don't hesitate to reach out.
 
Thank you for your understanding and continued support as we embark on this important transition.
Unfortunately for now we are not able to provide concrete timelines but we are currently working on a first preview version of the new WebComponent which should already offer a first impression of the concepts behind. We have gathered the feature requests and we surely have them in mind while developing the new component. 

Best Regards,
Hristo

**Comment by simlin** [comment]
Hi,

In the 2.0 version of the UI5 Web Components we introduced a new Table component:
https://sap.github.io/ui5-webcomponents/nightly/components/Table/

The table is still in an experimental phase and certain APIs might slightly change. Nevertheless we ask you to try out the new component already now. The earlier we get some feedback the better :)

We optimized the internal structure of the table, the usage of the ARIA role model and the keyboard support for a better accessibility.

Best Regards
Jens
